pkgs/development/libraries/pipewire/default.nix

```diff
@@ -69,13 +69,14 @@ let
 
   self = stdenv.mkDerivation rec {
     pname = "pipewire";
-    version = "0.3.54";
+    version = "0.3.55";
 
     outputs = [
       "out"
       "lib"
       "pulse"
       "jack"
+      "v4l2"
```

Comment:
Not sure about this.

Reply:
About what exactly?

Reply:
Any of this, I guess? I mean pw-v4l2 was built just fine even before this and it is in `$out` which gets added to systemPackages by the module already. All this seems to do is separate it into its own output for ??? reasons.

Reply:
I mostly did this to avoid the v4l2 dependency in case you don't want to use it. But it seems like the dependency is still there in the main executable so that didn't go as planned :(

Reply:
Removed the separate output.